environments.json

```diff
@@ -1283,7 +1283,7 @@
     "short": "SF 6.1,<br>SF 7",
     "obsolete": true
   },
-  "safari71_8": {
+  "safari7_1_8": {
```

Comment:
This is not Safari 7.1.8, but Safari 7.1 & Safari 8.  Also fixed on my patch on #1099